package.json

```diff
@@ -24,10 +24,10 @@
     "webpack": "webpack --config webpack.config.js"
   },
   "dependencies": {
-    "brq": "git+https://github.com/bucko13/brq.git#body-bug"
+    "brq": "git+https://github.com/bcoin-org/brq.git"
```

Comment:
This can be removed entirely now. This was just to enforce using the custom branch version of brq, but with the bclient updates and the packages published, it's no longer necessary